src/app/pages/BridgeWithdrawPage/components/SidebarSteps/index.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { useCallback, useMemo } from 'react';
```

Comment:
Can't we reuse `BridgeDepositPage` and just tweak it with some parameters (if necessary)? Because it looks like to that this is a duplicity.

Reply:
We decided not to do that and only share the ui component.